example/helloWorldLambda/src/helloWorldLambda.cpp

```diff
@@ -169,12 +167,11 @@ auto main()
     // This approach has the advantage that you do
     // not need to provide the signature of your function
     // as it is the case for the std::function example above.
-    auto const hiWorldBind (alpaka::exec::create<Acc> (
+    alpaka::kernel::exec<Acc> (
+        queue,
         workDiv,
-        std::bind( hiWorldFunction<Acc>, std::placeholders::_1, nExclamationMarks*3 )
-        ));
-
-    alpaka::queue::enqueue(queue, hiWorldBind);
+        std::bind( hiWorldFunction<Acc>, std::placeholders::_1, nExclamationMarks*2 )
```

Comment:
just a tiny detail, I guess `hiWorld...` could be replaced with `helloWorld` to be consistent with the example's title.

Reply:
I would change this the next time I have to change something in this file.